services/hsts/hsts.tester.js

```diff
@@ -0,0 +1,78 @@
+'use strict'
+
+const { ServiceTester } = require('..')
+const label = 'hsts'
+
+const t = (module.exports = new ServiceTester({
+  id: 'hsts',
+  title: 'HSTS Preload',
+}))
+
+t.create('gets the hsts status of github')
+  .get('/github.com.json')
+  .expectJSON({
+    name: label,
+    value: 'preloaded',
+  })
+
+t.create('gets the hsts status of httpforever')
+  .get('/httpforever.com.json')
+  .expectJSON({
+    name: label,
+    value: 'unknown',
+  })
+
+t.create('gets the status of an invalid uri')
+  .get('/does-not-exist.json')
+  .expectJSON({
+    name: label,
+    value: 'unknown',
+  })
+
+t.create('gets the hsts status of github (mock)')
+  .get('/github.com.json')
```

Comment:
Thanks for testing all paths! Another thing we should probably do here is validation of the custom colors. If you add a `style` query param with a value of `_shields_test` then the response will also include the applied color. For example:

`.get('github.com.json?style=_shields_test')`

Then below:

```
.expectJSON({
  name: label,
  value: 'preloaded',
  color: 'brightgreen'
})
```

Reply:
Thanks, I will add these shortly.